src/components/KonnectorInstall.jsx

```diff
@@ -1,27 +1,56 @@
-import React, { PureComponent } from 'react'
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
 
 import { TriggerManager } from 'cozy-harvest-lib'
 import flag from 'cozy-flags'
 
 import KonnectorMaintenance from './KonnectorMaintenance'
+import KonnectorSuccess from './KonnectorSuccess'
 import LegacyKonnectorInstall from './LegacyKonnectorInstall'
 import { getKonnector } from 'ducks/konnectors'
 import styles from '../styles/konnectorInstall'
 import { translate } from 'cozy-ui/react/I18n'
 
-export class KonnectorInstall extends PureComponent {
+export class KonnectorInstall extends Component {
```

Comment:
By curiosity, why did you switch from a PureComponent to a Component?

Reply:
As we handle the success from TriggerManager in this component, we do need now to keep a `state`, and `PureComponent` does not allow maintaining a state.

The advantage of a `PureComponent` is that it will be re-rendered only if its props have changed.

Reply:
> PureComponent does not allow maintaining a state.

You are sure about that?

https://reactjs.org/docs/react-api.html#reactpurecomponent says (emphasis is mine): 

> If your React component’s render() function renders the same result given the **same props and state**, you can use React.PureComponent for a performance boost in some cases.

Reply:
Yes I was sure. As strong as I am sure of the opposite now.

Thank you for pointing me this out (I amended the commit) !